node-tests/acceptance/end-to-end-test.js

```diff
@@ -97,7 +97,7 @@ describe('end-to-end', function() {
       return ember('new', 'ee-test-app').then(() => {
         process.chdir('ee-test-app');
 
-        return ember('install', `ember-electron@${path.join(packageTmpDir, 'package')}`);
+        return ember('install', `ember-electron@file:${path.join(packageTmpDir, 'ember-electron.tgz')}`);
```

Comment:
Yes, because npm >5 symlinks packages instead via file which was causing module resolution to fail. https://medium.com/@alex_young/npm-5-and-file-urls-3c3631f7367c

Reply:
Ah, gotcha. Then let's re-tar the package directory into a new temp tarball and use that so it picks up the version-cache-busting.

Reply:
@bendemboski I'm not sure how to do that. Is that something you or @stopdropandrew would be willing to do? (I'm done `git push -f`ing now -- fixed `.travis.yml`)

Reply:
I think something like this:

```javascript
return ember('new', 'ee-test-app').then(() => {
  process.chdir('ee-test-app');

  return run('tar', ['-czf', 'ember-electron-cachebust.tgz', '-C', packageTmpDir, 'package']);
}).then(() => {
  return ember('install', `ember-electron@file:${path.join(packageTmpDir, 'ember-electron-cachebust.tgz')}`);
});
```

Reply:
So we're talking about code [in `node-tests/acceptance/end-to-end-test.js`](https://github.com/jacobq/ember-electron/blob/support-ember-cli-3.2/node-tests/acceptance/end-to-end-test.js#L56), right?
Sorry, I think I'm not following you guys. Why do we need to make a second tarball instead of just using the first? Are they different?

Reply:
Yes, that's correct. The reason I think we need to do it is because [here](https://github.com/jacobq/ember-electron/blob/support-ember-cli-3.2/node-tests/acceptance/end-to-end-test.js#L53-L62) we package up the app, then untar it so we can change the version in `package.json`. If we don't do that, then it would have the same version as the latest released version of `ember-electron.

At least when I wrote this, `yarn` would look at the version in `package.json`, and if it had a cached version of `ember-electron` with the same version, it would use the cached version and ignore the tarball.

The cached version could be the latest release installed into some other node project somewhere from the npm registry, or a previous version of the local package, like if you're doing local development and run the tests, then make a change, then run the same tests. So it was very problematic.

Perhaps this behavior doesn't currently happen with `npm`, and maybe not even with `yarn` any more, but it still seems sketchy to install a package from a tarball whose version exactly matches that package's version in the npm registry -- just asking for some caching behavior to screw us.

So that's why after packaging it up, we untar it and change the version.

Reply:
OK, I think I get it now. We only use the extracted+modified contents of that tarball, but now we're using it directly, which may not be reliable due to potential cache issues.  I still have a couple questions though:
* Would you be opposed to this code being added in the same section where the package is packed, unpacked, and modified? (It makes sense to me to do it there even though one could argue that since it's only used in the `with npm` test right now we shouldn't do it until needed.)
* I know [`yarn pack` compresses the archive](https://yarnpkg.com/lang/en/docs/cli/pack/) and `.tgz`/`.tar.gz` is common, but since we're just making a temporary archive locally that we're going to immediately read back, I would prefer to just use `tar` ([`npm install` appears to support this](https://docs.npmjs.com/cli/install)) for performance / efficiency.